docs/source/esp-idf_component.rst

```diff
@@ -69,28 +96,41 @@ If you plan to base your code on examples provided in `examples <https://github.
 
     extern "C" void app_main()
     {
-        initArduino();
-        pinMode(4, OUTPUT);
-        digitalWrite(4, HIGH);
-        //do your own thing
-    }
+      initArduino();
+
+      // Arduino-like setup()
+      Serial.begin(115200);
+      while(!Serial){
+        ; // wait for serial port to connect
+      }
+
+      // Arduino-like loop()
+      while(true){
+        Serial.println("loop");
+      }
 
-- "Disable mutex locks for HAL"
-- If enabled, there will be no protection on the drivers from concurently accessing them from another thread/interrupt/core
-    - "Autoconnect WiFi on boot"
-- If enabled, WiFi will start with the last known configuration
-- Otherwise it will wait for WiFi.begin
+      // WARNING: if program reaches end of function app_main() the MCU will restart.
+    }
 
 Build, flash and monitor
 ************************
 
 - For both options use command ``idf.py -p <your-board-serial-port> flash monitor``
-- It will build, upload and open serial monitor to your board.
+
+  - The port is usually ``/dev/ttyUSB0`` search the active port with ``ls /dev/ttyUSB*``
```

Comment:
this is not true on Mac or Windows, Ports have very different names. It's maybe beyond this documentation to explain how one should find the serial port.

Reply:
Alright, I will remove this line in the next commit.